s3/src/main/java/ch/cyberduck/core/s3/S3Session.java

```diff
@@ -232,8 +233,8 @@ public void login(final Proxy proxy, final LoginCallback prompt, final CancelCal
                         new ThreadLocalHostnameDelegatingTrustManager(trust, host.getHostname()), key, prompt).configure(host);
             }
             // get temporary credentials for MinIO with Web Identity (OIDC)
-            else if (host.getProtocol().getOAuthAuthorizationUrl() != null) {
-                credentials = new NonAwsSTSCredentialsConfigurator(new ThreadLocalHostnameDelegatingTrustManager(trust,
+            else if(host.getProtocol().getOAuthAuthorizationUrl() != null) {
```

Comment:
The STS endpoint should be configurable in the profile as well and default to `https://sts.amazonaws.com`.